app/components/ListItem.js

```diff
@@ -63,6 +66,25 @@ export default class ListItem extends React.Component {
   }
 
   /**
+   * Makes check item call
+   */
+  handleCheck = () => {
+    this.setState({
```

Comment:
Should we still set the checkmark here? Even if an updateItem callback isn't passed?